app/javascript/dashboard/components/ChatList.vue

```diff
@@ -410,8 +410,10 @@ export default {
     },
     conversationListPagination() {
       const conversationsPerPage = 25;
+      const hasChatsOnView =
```

Comment:
@iamsivin Can you check if this is the same since arrays cannot be explicitly compared `this.chatsOnView !== []`

Reply:
@nithindavid This change will introduce the same bug there was before. https://linear.app/chatwoot/issue/CW-1424/pagination-not-working-properly